packages/blockchain-utils-validation/src/blockchains/__tests__/ethereum.test.ts

```diff
@@ -149,8 +152,8 @@ test.skip('invalid erc1271 proof should return null', async () => {
   await expect(ethereum.validateLink(erc1271Proof)).rejects.toThrow()
 })
 
-// TODO(NET-1107) ESM Mocking
-test.skip('validateLink: valid erc1271 proof should return proof', async () => {
+test('validateLink: valid erc1271 proof should return proof', async () => {
+  const ethereum = await import('../ethereum.js').then((m) => m.handler)
```

Comment:
why does this line look different than all the others?

Reply:
No difference, other than it was the first occurrence, thus not quite idiomatic. Fixed.